docker-entrypoint-initdb.d/install_timescaledb.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+TS_TELEMETRY='basic'
```

Comment:
can't this just be:
`TS_TELEMETRY=${TS_TELEMETRY:-'basic'}`

Reply:
Wellllllll, I want to prevent cases where `TS_TELEMETRY` is set to some random string and then that junk gets written to the conf file.

Reply:
AH I see, okay leave as is.